zebrad/Cargo.toml

```
@@ -89,6 +89,7 @@ elasticsearch = [
 sentry = ["dep:sentry"]
 journald = ["tracing-journald"]
 filter-reload = ["hyper", "http-body-util", "hyper-util", "bytes"]
+health-endpoint = ["hyper", "hyper-util", "serde/derive", "chrono/serde", "serde_json"]
```

```suggestion
health-endpoint = ["hyper", "hyper-util", "hyper-util/tokio", "serde/derive", "chrono/serde", "serde_json"]
```
Consider adding "hyper-util/tokio" to the list because `cargo check --features=health-endpoint` currently fails with:

```
error[E0433]: failed to resolve: could not find `TokioIo` in `rt`
  --> zebrad/src/components/health.rs:45:38
   |
45 |             let io = hyper_util::rt::TokioIo::new(stream);
   |                                      ^^^^^^^ could not find `TokioIo` in `rt`
```